components/server/src/setup.rs

```diff
@@ -34,12 +35,22 @@ macro_rules! fatal {
 // TODO: There is a very small chance that duplicate files will be generated if there are
 // a lot of logs written in a very short time. Consider rename the rotated file with a version
 // number while rotate by size.
+//
+// The file name format after rotated is as follows: tikv-2021-12-21T00-19-38.378.log
```

Comment:
The format is not unified with comments above `RotatingFileLogger`
> /// After rotating, the original log file would be renamed to "{original name}.{%Y-%m-%d-%H:%M:%S}".

Reply:
Fixed thanks!